tests/test_cli.py

```diff
@@ -407,18 +408,18 @@ async def test_record(self):
         subset_urls = list(map(lambda record: record.key, subset))
         results = await Predict.cli(
             "record",
-            "-sources",
-            "primary=json",
-            "-source-filename",
-            self.temp_filename,
+            "-keys",
```

Comment:
`-source-filename` was not taking anything and was being passed as a flag resulting in the PosixPath('True') error.

A few optional args were changed to positional args so that's why we are facing these errors. Need to find a better approach for positional args I guess.

Reply:
Nice work figuring this out!